models/models.js

```diff
@@ -87,7 +87,7 @@ exports.fetchReviewCommentsById = (review_id) => {
       return comments.rows;
     });
 };
-
+//task 7
 exports.insertCommentsByReviewId = (review_id, commentBody) => {
   return checkReviewIdExists(review_id)
     .then(() => {
```

Comment:
on line 99, this is a very inefficient way of checking whether the required keys are on your comment object. instead, i would just check whether comment.username and comment.body are truthy!
this is a valid way of doing it though - just to give some food for thought